public/data/PrivacyPolicy.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Privacy Policy
+---
+
+Privasi Anda penting bagi kami. Merupakan kebijakan **Coronator** untuk menghormati privasi Anda mengenai informasi apa pun yang dapat kami kumpulkan dari Anda di situs web kami, [https://coronator.id/](https://coronator.id/), dan situs lain yang kami miliki dan operasikan.
+<br><br>
+Kami hanya meminta informasi pribadi ketika kami benar-benar membutuhkannya untuk memberikan layanan kepada Anda. Kami mengumpulkannya dengan cara yang adil dan sah, dengan sepengetahuan dan persetujuan Anda. Kami juga memberi tahu Anda mengapa kami mengumpulkannya dan bagaimana penggunaannya.
+<br><br>
+Kami hanya menyimpan informasi yang dikumpulkan selama diperlukan untuk menyediakan layanan yang Anda minta. Data apa yang kami simpan, akan kami lindungi dalam cara yang dapat diterima secara komersial untuk mencegah kehilangan dan pencurian, serta akses, pengungkapan, penyalinan, penggunaan, atau modifikasi yang tidak sah.
```

Comment:
Ini maksudnya gimana Jan?
`kami lindungi dalam cara yang dapat diterima secara komersial`

Reply:
We only retain collected information for as long as necessary to provide you with your requested service. What data we store, we’ll protect within commercially acceptable means to prevent loss and theft, as well as unauthorized access, disclosure, copying, use or modification.

seperti itu aslinya rib